home_work_1/task1/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('homepage/', views.homepage, name='homepage'),
+    path('home/', views.homepage, name='home'),
+    path('', views.homepage, name='slash'),
```

Comment:
```suggestion
    path('', views.homepage, name='homepage'),
```
А з інших можна прибрати. Навряд ти будеш звертатись до них через `reverse` або тег `{% url %}`.

Reply:
Прибрав) ДЯкую за ревью)